### ISSUE TITLE
Doesn't work on Android phone

### ISSUE THREAD

**Issue by greenzorro** [bingryan/obsidian-markdown-export-plugin#48]
OnePlus 6T, Android 10, to be specific. 

I see export to markdown/HTML in the Obsidian Android app. They do nothing after being clicked.

Not sure if it's a bug or just the way it is that this plugin works only on PC and Mac.

**Comment by bingryan** [comment]
The mobile has not been tested :)

**Comment by josephtribulat** [comment]
It doesn't seem to work on an iPad either.
Too bad, as it seemed to be the only export plug-in available on the mobile version. Looking forward to testing the desktop version at home.

**Comment by acfogarty** [comment]
Yes, unfortunately this otherwise great and useful plugin will not work on mobile because it uses Node.js (see doc at https://docs.obsidian.md/Plugins/Getting+started/Mobile+development)

@bingryan would you consider setting isDesktopOnly to true in your manifest.json to avoid people mistakenly trying to install it on mobile?

**Comment by bingryan** [comment]
> Yes, unfortunately this otherwise great and useful plugin will not work on mobile because it uses Node.js (see doc at https://docs.obsidian.md/Plugins/Getting+started/Mobile+development)
> 
> @bingryan would you consider setting isDesktopOnly to true in your manifest.json to avoid people mistakenly trying to install it on mobile?

sure :)
